src/components/common/ModalComponent.jsx

```diff
@@ -12,20 +13,24 @@ export default class ArtModal extends Component {
     const { children } = this.props;
     const childrenWithProps = React.Children.map(children, child =>
       React.cloneElement(child,
-        { toggleModal: this.toggleModal }
+        { togglemodal: this.toggleModal }
```

Comment:
Changing this to lower case will affect multiple code that uses the modal. Is there any particular reason for the change ?

Reply:
@HawiCaesar it was a mistake, I'll fix it thanks.